apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -74,15 +75,20 @@ handle_request('GetBlockByHash' = _Method, Req, _Context) ->
                     %% or map and always runs jsx:encode/1 on it - even if it
                     %% is already encoded to a binary; that's why we use
                     %% aec_blocks:serialize_to_map/1 instead of
-                    %% aec_blocks:serialize_for_network/1 
+                    %% aec_blocks:serialize_for_network/1
+                    lager:debug("Block = ~p", [Block]),
                     Resp =
                       cleanup_genesis(aec_blocks:serialize_to_map(Block)),
+                    lager:debug("Resp = ~p", [Resp]),
                     {200, [], Resp};
                 {error, {block_not_found, _}} ->
                     {404, [], #{reason => <<"Block not found">>}}
             end
     end;
 
+handle_request('GetTxs', _Req, _Context) ->
+    error(nyi);
```

Comment:
Is there a ticket tracking implementation of this API?
(I am concerned that this API would allow a small request to receive a huge response.)

Reply:
That's an issue I brought up earlier, put then decided to punt on. A problem with a chunking request (which could be added via an optional parameter and a continuation method) is that we'd need to define a key hash value for transactions in the pool. This, in its turn, depends on the sorting algorithm, which @sennui was working on in parallel.

But I agree that there should be a ticket. It's just been almost impossible to get this merged since the task was quite broad to begin with. I suggest we create that ticket and deal with it separately.

Reply:
Sorry, it's not just a question of the chunking (or not), but the fact that it isn't implemented at all. :)
My plan was to address that as soon as I get this merged.

Reply:
Yes, no need to implement in this branch. Thanks.

Reply:
Created PT ticket #152714484 regarding the chunking.